### DISCUSSION TITLE
How to change a card's bgcolor?

### DISCUSSION THREAD

**Discussion by Haadi-A** [flet-dev/flet#972] [Q&A]
Is there a way to change the bg colour of the Card control cuz I'm trying to use it as a neumorphic tile and its colour is supposed to be white not the default blue, so Im wondering whether there is a way I could change its colour or even if there is an alternative to the Card which would allow it's bgcolor to be changed as well as accepting other controls as its children or child , as I didn't see any bgcolor or color parameter in the Card's list of parameters.

**Comment by FeodorFitsner** [comment]
https://github.com/flet-dev/flet/issues/973
